frontend/pages/hosts/details/DeviceUserPage/components/SettingUpYourDevice/SettingUpYourDevice.tsx

```diff
@@ -75,7 +77,9 @@ const SettingUpYourDevice = ({
       <Card borderRadiusSize="xxlarge" paddingSize="xlarge">
         <div className={`${baseClass}__header`}>
           <h2>{title}</h2>
-          {!failedSoftware && <InfoButton onClick={toggleInfoModal} />}
+          {!failedSoftware && platform !== "darwin" && (
+            <InfoButton onClick={toggleInfoModal} />
+          )}
```

Comment:
Confirmed with product that we're ok 🪓 ing this modal for macOS setup experience.